include/colors

```diff
@@ -11,15 +11,15 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
-if [[ $MODE != "mono" && $MODE != "text" && $MODE != "csv" && $MODE != "json" ]]; then
+if [[ "$MODE" != "mono" && "$MODE" != "text" && "$MODE" != "csv" && "$MODE" != "json" && "$MODE" != "json-asff" && "$MODE" != "securityhub" ]]; then
```

Comment:
We can add a new option like `-S` (capital S) and move the securityhub integration to another function. Using `json-asff` as output and if `-S` is used then send information to SH.

Reply:
Thanks for the feedback the other day, this has been addressed